Codecademy/gamut
# refactor(ContentGroupBaseCard): CourseDifficulty export and description truncation

### Overview

- Fixed exporting for the types in ContentGroupBaseCard
- Changed description truncation to 3 per design feedback 





### PR Checklist


#### Testing Instructions

Check out the mono env and see existing cards are the same aside from those with longer descriptions should now be 3 lines not 4. In addition, all tests should pass despite new export fix. No visible changes on monolith yet. 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34642)
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/1260)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1260)
